src/ResourceManager/DataLakeAnalytics/DataLake.InternalAnalytics.Scope/DataLake.InternalAnalytics.Scope.csproj

```diff
@@ -0,0 +1,84 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this project expected to be here? This should be part of SDK right? Are we intentionally coupling SDK and Powershell releases? ...Next time we need to update the SDK , we will have to go through the external public powershell release cycle